### ISSUE TITLE
PLEASE IMPROVE THE DOCS!

### ISSUE THREAD

**Issue by trolliama** [futurepress/epub.js#1146]
It's so hard to understand the framework with a poor doc, i am really confused about everything.

**Comment by mcthulhu** [comment]
There are several projects using it that might help for comparison. Where are you trying to implement it? Desktop, browser, ...?

**Comment by webservant316** [comment]
I would like to see improvement to the docs as well.  Thanks for whatever improvements can be made.

**Comment by Navds** [comment]
Same, spent half a day just to get my epub rendered on the browser. Didn't know that you need to define global.ePub, didn't know that you have to add navigation manually, stuck with the cover displayed statically until I found the css at https://github.com/futurepress/epub.js/tree/master/examples
Oh yeah, even the npm install got me confused, didn't know the official package name.

**Comment by trolliama** [comment]
I would like to implement on the desktop. The problem with your documents is that you just put the method or object, but don't tell us what it is and what to do with it. You just give the name of the method and what it returns, but don't say what it is for, you know? And some epub things like epubcfi must have some sort of explanation, even if a little bit just for the reader to situate themselves

**Like this:**
![image](https://user-images.githubusercontent.com/21108976/100300516-f0f17580-2f74-11eb-92ba-7dafedd4b51f.png)
Ok ... but what exactly is a location? what is this pause attribute and request? 

 And the main thing: examples. I would really appreciate if the docs had more examples, its much easier to understand

**Comment by trolliama** [comment]
This was just an example. I think this is a good api, but it’s so hard to learn because of the poor docs

**Comment by trolliama** [comment]
Sorry, i am trying on the browser for a while

**Comment by johnfactotum** [comment]
Yeah, the documentation could really use some explanations and examples. Though, to be fair, in this specific example, if you filter for "locations" in the documentation, you'd see that `Book.locations` returns a `Locations` object, so one can reasonably assume that that is how you'd access it. Since you receive a ready-made `Locations` object, one has no use for the parameters `spine`, `request`, etc., which are only used when constructing the object.

The docs can probably be improved immensely by simply linking to the relevant example files; in this case https://github.com/futurepress/epub.js/blob/master/examples/locations.html will show you how to use `Locations`.
